aws-solutions/serverless-image-handler
# Remove html from demo-ui text entry fields

**Description of changes:**
The demo-ui contains bucket and key text fields that are processed as is. If html tags are included in the text fields, unexpected behavior may occur when the image is imported and previewed.

This PR includes changes to ensure that the demo-ui bucket and key text fields are scrubbed of potential html.

Tests performed using demo-ui:

* import/preview image with bucket/key containing text - confirmed that image loads as expected
* import/preview image with bucket containing html with embedded text - confirmed that html is removed and image loads as expected
* import/preview image with key containing html with embedded text - confirmed that html is removed and image loads as expected
* import/preview image with key containing html with embedded unicode text - confirmed that html is removed and image loads as expected
* in each case, Request Body box confirmed that request contains only the bucket and key text
* confirmed that browser console did not include any new error messages or warnings

**Checklist**
- [ ] :wave: I have added unit tests for all code changes.
- [x] :wave: I have run the unit tests, and all unit tests have passed.
- [ ] :warning: This pull request might incur a breaking change.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
